bin/plotting/pycbc_page_snrchi

```diff
@@ -13,40 +16,77 @@ parser.add_argument('--segment-name', default=None, type=str,
                     help='Optional, name of segment list to use for vetoes')
 parser.add_argument('--min-snr', type=float, help='Optional, Minimum SNR to plot')
 parser.add_argument('--output-file')
-parser.add_argument('--newsnr-contours', nargs='*', help="List of newsnr values to draw contours at.", default=[])
+parser.add_argument(
+    '--newsnr-contours',
+    nargs='*',
+    help="List of newsnr values to draw contours at. Only valid for "
+         "--chisq-choice traditional (which is default)"
+)
 parser.add_argument('--chisq-choice', choices=chisq_choices,
                     default='traditional',
                     help='Which chisquared to plot. Default=traditional')
 args = parser.parse_args()
 
-f = h5py.File(args.trigger_file, 'r')
-ifo = tuple(f.keys())[0]
-f = f[ifo]
-snr = f['snr'][:]
-chisq = get_chisq_from_file_choice(f, args.chisq_choice)
+pycbc.init_logging(1)
+
+if args.newsnr_contours is not None and not args.chisq_choice == 'traditional':
+    parser.error(
+        "Newsnr contours are being plotted assuming a traditional "
+        "chisquared, but this is not being plotted! We are plotting %s" %
+        args.chisq_choice
+    )
+
+args.newsnr_contours = \
+    args.newsnr_contours if args.newsnr_contours is not None else []
+
+# First - get the IFO from the file
+with h5py.File(args.trigger_file, 'r') as f:
+    ifo = tuple(f.keys())[0]
+
+logging.info(
+    "Plotting SNR vs %s chisq for %s triggers",
+    args.chisq_choice,
+    ifo
+)
+
+if args.min_snr is not None:
+    filter_rank = 'snr'
+else:
+    filter_rank = None
+    logging.warning(
```

Comment:
Perhaps you could only trigger this warning if the number of triggers in the file is large? It's quick to get the length of the SNR array.

Reply:
I've added a warning for 1e7 triggers or more - that may be too low, but its better than nothing